app/src/main/java/woowacourse/movie/activity/SeatPickerActivity.kt

```diff
@@ -0,0 +1,238 @@
+                )
+            }
+            setPickDoneButtonColor()
+
+            Toast.makeText(this, "$seatNames", Toast.LENGTH_SHORT).show()
+            getSeats().forEach {
+                if (seatNames.contains(it.text)) it.setBackgroundColor(getColor(R.color.picked_seat_color))
+            }
+        }
+    }
+
+    private fun setPickDoneButtonClickListener() {
+        pickDoneButton.setOnClickListener {
+            AlertDialog.Builder(this)
+                .setTitle(getString(R.string.alert_dialog_book_confirm))
+                .setMessage(getString(R.string.alert_dialog_book_re_confirm))
+                .setPositiveButton(
+                    getString(R.string.alert_dialog_book_done)
+                ) { _, _ ->
+                    val intent = BookCompleteActivity.intent(this)
+                    intent.putExtra(
+                        BundleKeys.MOVIE_BOOKING_SEAT_INFO_KEY,
+                        MovieBookingSeatInfo(
+                            getMovieBookingInfo(),
+                            seatGroup.sorted().seats.map {
+                                formatSeatName(it.row.value, it.column.value)
+                            },
+                            seatPickerTicketPrice.text.toString()
+                        )
+                    )
+                    startActivity(intent)
+                }.setNegativeButton(
+                    getString(R.string.alert_dialog_book_cancel)
+                ) { dialog, _ ->
+                    dialog.dismiss()
+                }
+                .setCancelable(false)
+                .show()
+        }
+    }
+
+    private fun setSeatsOnClickListener(movieBookingInfo: MovieBookingInfo) {
+        getSeats().forEachIndexed { index, seat ->
+            seat.setOnClickListener {
+                val newSeat = Seat(SeatRow(index / 4), SeatColumn(index % 4))
+                if (seatGroup.seats.contains(newSeat)) {
+                    progressRemoveSeat(newSeat, seat, movieBookingInfo)
+                    return@setOnClickListener
+                }
+                if (seatGroup.canAdd(ticketBundle.count) && !seatGroup.seats.contains(newSeat)) {
+                    progressAddSeat(newSeat, seat, movieBookingInfo)
+                }
+            }
+        }
+    }
+
+    private fun progressAddSeat(
+        newSeat: Seat,
+        seat: TextView,
+        movieBookingInfo: MovieBookingInfo,
+    ) {
+        seatGroup = seatGroup.add(newSeat)
+        ticketBundle = ticketBundle.add(Ticket(newSeat.getSeatTier().price))
+        seat.setBackgroundColor(getColor(R.color.picked_seat_color))
+        setPickDoneButtonColor()
+        seatPickerTicketPrice.text =
+            getString(R.string.ticket_price_format)
+                .format(
+                    ticketBundle.calculateTotalPrice(
+                        movieBookingInfo.date,
+                        movieBookingInfo.time
+                    )
+                )
```

Comment:
```suggestion
            getString(
                R.string.ticket_price_format,
                ticketBundle.calculateTotalPrice(
                    movieBookingInfo.date,
                    movieBookingInfo.time
                )
            )
```
String.format 함수를 잘 활용해주셨습니다. 👍 
다만, 안드로이드의 getString은 기본적으로 format을 지원합니다. (이하 관련 내용 동일)